### PR TITLE
LookupTable: Support setting table in constructor

### LINKED ISSUE #1257: can't deserialize vtkLookupTable correctly
Hello,

From the example https://kitware.github.io/vtk-js/examples/ActorSerialization.html,
I tried to change the `table` of the lookupTable. however it seems that at deserialization of the object the table is reseted by the default one.
I changed:
```
"table": [255.5, 0.5, 0.5, 255.5, 255.5, 4.50002, 0.5, 255.5, 255.5, 8.50004, 0.5, 255.5, 255.5, 12.50006, 0.5, 255.5, 255.5, 16.50008, 0.5, 255.5, 255.5, 20.5001, 0.5, 255.5, 255.5, 24.50012, 0.5, 255.5, 255.5, 28.500140000000002, 0.5, 255.5, 255.5, 32.50016, 0.5, 255.5, 255.5, 36.50018, 0.5, 255.5, 255.5, 40.5002, 0.5, 255.5, 255.5, 44.500220000000006, 0.5, 255.5, 255.5, 48.50024, 0.5, 255.5, 255.5, 52.500260000000004, 0.5, 255.5, 255.5, 56.500280000000004, 0.5, 255.5, 255.5, 60.5003, 0.5, 255.5, 255.5, 64.50032, 0.5, 255.5, 255.5, 68.50034000000001, 0.5, 255.5, 255.5, 72.50036, 0.5, 255.5, 255.5, 76.50038, 0.5, 255.5, 255.5, 80.5004, 0.5, 255.5, 255.5, 84.50042, 0.5, 255.5, 255.5, 88.50044000000001, 0.5, 255.5, 255.5, 92.50046000000002, 0.5, 255.5, 255.5, 96.50048, 0.5, 255.5, 255.5, 100.50049999999999, 0.5, 255.5, 255.5, 104.50052000000001, 0.5, 255.5, 255.5, 108.50054000000002, 0.5, 255.5, 255.5, 112.50056000000001, 0.5, 255.5, 255.5, 116.50057999999999, 0.5, 255.5, 255.5, 120.5006, 0.5, 255.5, 255.5, 124.50062000000001, 0.5, 255.5, 255.5, 128.50064, 0.5, 255.5, 255.5, 132.50066, 0.5, 255.5, 255.5, 136.50068000000002, 0.5, 255.5, 255.5, 140.50070000000002, 0.5, 255.5, 255.5, 144.50072, 0.5, 255.5, 255.5, 148.50073999999998, 0.5, 255.5, 255.5, 152.50076, 0.5, 255.5, 255.5, 156.50078000000002, 0.5, 255.5, 255.5, 160.5008, 0.5, 255.5, 255.5, 164.50082, 0.5, 255.5, 255.5, 168.50084, 0.5, 255.5, 255.5, 172.50086000000002, 0.5, 255.5, 255.5, 176.50088000000002, 0.5, 255.5, 255.5, 180.5009, 0.5, 255.5, 255.5, 184.50092000000004, 0.5, 255.5, 255.5, 188.50094, 0.5, 255.5, 255.5, 192.50096, 0.5, 255.5, 255.5, 196.50098, 0.5, 255.5, 255.5, 200.50099999999998, 0.5, 255.5, 255.5, 204.50102, 0.5, 255.5, 255.5, 208.50104000000002, 0.5, 255.5, 255.5, 212.50106, 0.5, 255.5, 255.5, 216.50108000000003, 0.5, 255.5, 255.5, 220.50110000000004, 0.5, 255.5, 255.5, 224.50112000000001, 0.5, 255.5, 255.5, 228.50114, 0.5, 255.5, 255.5, 232.50115999999997, 0.5, 255.5, 255.5, 236.50118, 0.5, 255.5, 255.5, 240.5012, 0.5, 255.5, 255.5, 244.50122, 0.5, 255.5, 255.5, 248.50124000000002, 0.5, 255.5, 255.5, 252.50126000000003, 0.5, 255.5, 254.49871999999996, 255.5, 0.5, 255.5, 250.49869999999999, 255.5, 0.5, 255.5, 246.49867999999998, 255.5, 0.5, 255.5, 242.49865999999997, 255.5, 0.5, 255.5, 238.49864, 255.5, 0.5, 255.5, 234.49862, 255.5, 0.5, 255.5, 230.49859999999995, 255.5, 0.5, 255.5, 226.49857999999998, 255.5, 0.5, 255.5, 222.49855999999997, 255.5, 0.5, 255.5, 218.49854, 255.5, 0.5, 255.5, 214.49851999999998, 255.5, 0.5, 255.5, 210.49849999999995, 255.5, 0.5, 255.5, 206.49847999999997, 255.5, 0.5, 255.5, 202.49845999999997, 255.5, 0.5, 255.5, 198.49843999999996, 255.5, 0.5, 255.5, 194.49841999999998, 255.5, 0.5, 255.5, 190.49839999999998, 255.5, 0.5, 255.5, 186.49838, 255.5, 0.5, 255.5, 182.49836, 255.5, 0.5, 255.5, 178.49833999999996, 255.5, 0.5, 255.5, 174.49831999999998, 255.5, 0.5, 255.5, 170.49829999999997, 255.5, 0.5, 255.5, 166.49827999999994, 255.5, 0.5, 255.5, 162.49825999999996, 255.5, 0.5, 255.5, 158.49823999999998, 255.5, 0.5, 255.5, 154.49822, 255.5, 0.5, 255.5, 150.4982, 255.5, 0.5, 255.5, 146.49817999999996, 255.5, 0.5, 255.5, 142.49815999999998, 255.5, 0.5, 255.5, 138.49813999999998, 255.5, 0.5, 255.5, 134.49811999999994, 255.5, 0.5, 255.5, 130.49809999999997, 255.5, 0.5, 255.5, 126.49807999999997, 255.5, 0.5, 255.5, 122.49805999999994, 255.5, 0.5, 255.5, 118.49803999999999, 255.5, 0.5, 255.5, 114.49801999999995, 255.5, 0.5, 255.5, 110.498, 255.5, 0.5, 255.5, 106.49797999999998, 255.5, 0.5, 255.5, 102.49795999999995, 255.5, 0.5, 255.5, 98.49794, 255.5, 0.5, 255.5, 94.49791999999997, 255.5, 0.5, 255.5, 90.49789999999993, 255.5, 0.5, 255.5, 86.49788, 255.5, 0.5, 255.5, 82.49785999999996, 255.5, 0.5, 255.5, 78.49783999999993, 255.5, 0.5, 255.5, 74.49781999999998, 255.5, 0.5, 255.5, 70.49779999999994, 255.5, 0.5, 255.5, 66.49778, 255.5, 0.5, 255.5, 62.49775999999997, 255.5, 0.5, 255.5, 58.497739999999936, 255.5, 0.5, 255.5, 54.49771999999999, 255.5, 0.5, 255.5, 50.49769999999996, 255.5, 0.5, 255.5, 46.49768000000001, 255.5, 0.5, 255.5, 42.497659999999975, 255.5, 0.5, 255.5, 38.49763999999995, 255.5, 0.5, 255.5, 34.49762, 255.5, 0.5, 255.5, 30.497599999999967, 255.5, 0.5, 255.5, 26.497579999999935, 255.5, 0.5, 255.5, 22.49755999999999, 255.5, 0.5, 255.5, 18.497539999999958, 255.5, 0.5, 255.5, 14.497519999999925, 255.5, 0.5, 255.5, 10.497499999999977, 255.5, 0.5, 255.5, 6.497479999999946, 255.5, 0.5, 255.5, 2.4974599999999993, 255.5, 0.5, 255.5, 0.5, 255.5, 2.5025600000000328, 255.5, 0.5, 255.5, 6.502580000000065, 255.5, 0.5, 255.5, 10.502600000000012, 255.5, 0.5, 255.5, 14.502620000000043, 255.5, 0.5, 255.5, 18.50263999999999, 255.5, 0.5, 255.5, 22.502660000000024, 255.5, 0.5, 255.5, 26.502680000000055, 255.5, 0.5, 255.5, 30.5027, 255.5, 0.5, 255.5, 34.50272000000003, 255.5, 0.5, 255.5, 38.50274000000007, 255.5, 0.5, 255.5, 42.50276000000001, 255.5, 0.5, 255.5, 46.502780000000044, 255.5, 0.5, 255.5, 50.50280000000008, 255.5, 0.5, 255.5, 54.50282000000002, 255.5, 0.5, 255.5, 58.502840000000056, 255.5, 0.5, 255.5, 62.50286, 255.5, 0.5, 255.5, 66.50288000000003, 255.5, 0.5, 255.5, 70.50290000000007, 255.5, 0.5, 255.5, 74.50292000000002, 255.5, 0.5, 255.5, 78.50294000000004, 255.5, 0.5, 255.5, 82.50295999999999, 255.5, 0.5, 255.5, 86.50298000000002, 255.5, 0.5, 255.5, 90.50300000000006, 255.5, 0.5, 255.5, 94.50302, 255.5, 0.5, 255.5, 98.50304000000003, 255.5, 0.5, 255.5, 102.50306000000006, 255.5, 0.5, 255.5, 106.50308000000001, 255.5, 0.5, 255.5, 110.50310000000005, 255.5, 0.5, 255.5, 114.50312000000008, 255.5, 0.5, 255.5, 118.50314000000002, 255.5, 0.5, 255.5, 122.50316000000005, 255.5, 0.5, 255.5, 126.50318, 255.5, 0.5, 255.5, 130.50320000000002, 255.5, 0.5, 255.5, 134.50322000000006, 255.5, 0.5, 255.5, 138.50324, 255.5, 0.5, 255.5, 142.50326000000004, 255.5, 0.5, 255.5, 146.50328, 255.5, 0.5, 255.5, 150.50330000000002, 255.5, 0.5, 255.5, 154.50332000000006, 255.5, 0.5, 255.5, 158.50334, 255.5, 0.5, 255.5, 162.50336000000004, 255.5, 0.5, 255.5, 166.50338000000008, 255.5, 0.5, 255.5, 170.5034, 255.5, 0.5, 255.5, 174.50342000000003, 255.5, 0.5, 255.5, 178.50344000000007, 255.5, 0.5, 255.5, 182.50346000000002, 255.5, 0.5, 255.5, 186.50348000000005, 255.5, 0.5, 255.5, 190.5035, 255.5, 0.5, 255.5, 194.50352000000004, 255.5, 0.5, 255.5, 198.50354000000007, 255.5, 0.5, 255.5, 202.50356000000002, 255.5, 0.5, 255.5, 206.50358000000006, 255.5, 0.5, 255.5, 210.5036, 255.5, 0.5, 255.5, 214.50362, 255.5, 0.5, 255.5, 218.50364000000005, 255.5, 0.5, 255.5, 222.50366, 255.5, 0.5, 255.5, 226.50368000000003, 255.5, 0.5, 255.5, 230.50370000000007, 255.5, 0.5, 255.5, 234.50372000000002, 255.5, 0.5, 255.5, 238.50374000000005, 255.5, 0.5, 255.5, 242.50376000000009, 255.5, 0.5, 255.5, 246.50378000000003, 255.5, 0.5, 255.5, 250.50380000000007, 255.5, 0.5, 255.5, 254.50382, 255.5, 0.5, 252.49615999999995, 255.5, 255.5, 0.5, 248.4961399999999, 255.5, 255.5, 0.5, 244.49611999999988, 255.5, 255.5, 0.5, 240.4961, 255.5, 255.5, 0.5, 236.49607999999998, 255.5, 255.5, 0.5, 232.49605999999994, 255.5, 255.5, 0.5, 228.4960399999999, 255.5, 255.5, 0.5, 224.49601999999987, 255.5, 255.5, 0.5, 220.496, 255.5, 255.5, 0.5, 216.49598, 255.5, 255.5, 0.5, 212.49595999999997, 255.5, 255.5, 0.5, 208.49593999999993, 255.5, 255.5, 0.5, 204.4959199999999, 255.5, 255.5, 0.5, 200.49589999999986, 255.5, 255.5, 0.5, 196.49588, 255.5, 255.5, 0.5, 192.49585999999996, 255.5, 255.5, 0.5, 188.49583999999993, 255.5, 255.5, 0.5, 184.4958199999999, 255.5, 255.5, 0.5, 180.49579999999986, 255.5, 255.5, 0.5, 176.49578000000002, 255.5, 255.5, 0.5, 172.49576, 255.5, 255.5, 0.5, 168.49573999999996, 255.5, 255.5, 0.5, 164.49571999999992, 255.5, 255.5, 0.5, 160.49569999999989, 255.5, 255.5, 0.5, 156.49567999999985, 255.5, 255.5, 0.5, 152.49566, 255.5, 255.5, 0.5, 148.49563999999995, 255.5, 255.5, 0.5, 144.49561999999992, 255.5, 255.5, 0.5, 140.49559999999988, 255.5, 255.5, 0.5, 136.49557999999985, 255.5, 255.5, 0.5, 132.49556, 255.5, 255.5, 0.5, 128.49553999999995, 255.5, 255.5, 0.5, 124.49551999999994, 255.5, 255.5, 0.5, 120.49549999999991, 255.5, 255.5, 0.5, 116.49547999999987, 255.5, 255.5, 0.5, 112.49546000000001, 255.5, 255.5, 0.5, 108.49543999999997, 255.5, 255.5, 0.5, 104.49541999999995, 255.5, 255.5, 0.5, 100.49539999999992, 255.5, 255.5, 0.5, 96.49537999999988, 255.5, 255.5, 0.5, 92.49536000000002, 255.5, 255.5, 0.5, 88.49533999999998, 255.5, 255.5, 0.5, 84.49531999999995, 255.5, 255.5, 0.5, 80.49529999999993, 255.5, 255.5, 0.5, 76.4952799999999, 255.5, 255.5, 0.5, 72.49525999999986, 255.5, 255.5, 0.5, 68.49524, 255.5, 255.5, 0.5, 64.49521999999996, 255.5, 255.5, 0.5, 60.49519999999993, 255.5, 255.5, 0.5, 56.495179999999905, 255.5, 255.5, 0.5, 52.49515999999987, 255.5, 255.5, 0.5, 48.495140000000006, 255.5, 255.5, 0.5, 44.49511999999998, 255.5, 255.5, 0.5, 40.495099999999944, 255.5, 255.5, 0.5, 36.495079999999916, 255.5, 255.5, 0.5, 32.49505999999988, 255.5, 255.5, 0.5, 28.49503999999985, 255.5, 255.5, 0.5, 24.49501999999999, 255.5, 255.5, 0.5, 20.494999999999955, 255.5, 255.5, 0.5, 16.494979999999924, 255.5, 255.5, 0.5, 12.494959999999892, 255.5, 255.5, 0.5, 8.494939999999861, 255.5, 255.5, 0.5, 4.494919999999999, 255.5, 255.5, 0.5051000000000334, 0.5, 255.5, 255.5, 255.5, 0.5, 0.5, 255.5, 0.5051000000000334, 0.5, 255.5, 255.5, 128, 0.5, 0.5, 255.5],
```
by 
```
"table": [68.5, 1.5, 84.5, 255.5, 68.5, 2.5, 85.5, 255.5, 68.5, 3.5, 87.5, 255.5, 69.5, 5.5, 88.5, 255.5, 69.5, 6.5, 90.5, 255.5, 69.5, 8.5, 91.5, 255.5, 70.5, 9.5, 92.5, 255.5, 70.5, 11.5, 94.5, 255.5, 70.5, 12.5, 95.5, 255.5, 70.5, 14.5, 97.5, 255.5, 71.5, 15.5, 98.5, 255.5, 71.5, 17.5, 99.5, 255.5, 71.5, 18.5, 101.5, 255.5, 71.5, 20.5, 102.5, 255.5, 71.5, 21.5, 103.5, 255.5, 71.5, 22.5, 105.5, 255.5, 71.5, 24.5, 106.5, 255.5, 72.5, 25.5, 107.5, 255.5, 72.5, 26.5, 108.5, 255.5, 72.5, 28.5, 110.5, 255.5, 72.5, 29.5, 111.5, 255.5, 72.5, 30.5, 112.5, 255.5, 72.5, 32.5, 113.5, 255.5, 72.5, 33.5, 114.5, 255.5, 72.5, 34.5, 115.5, 255.5, 72.5, 35.5, 116.5, 255.5, 71.5, 37.5, 117.5, 255.5, 71.5, 38.5, 118.5, 255.5, 71.5, 39.5, 119.5, 255.5, 71.5, 40.5, 120.5, 255.5, 71.5, 42.5, 121.5, 255.5, 71.5, 43.5, 122.5, 255.5, 71.5, 44.5, 123.5, 255.5, 70.5, 45.5, 124.5, 255.5, 70.5, 47.5, 124.5, 255.5, 70.5, 48.5, 125.5, 255.5, 70.5, 49.5, 126.5, 255.5, 69.5, 50.5, 127.5, 255.5, 69.5, 52.5, 127.5, 255.5, 69.5, 53.5, 128.5, 255.5, 69.5, 54.5, 129.5, 255.5, 68.5, 55.5, 129.5, 255.5, 68.5, 57.5, 130.5, 255.5, 67.5, 58.5, 131.5, 255.5, 67.5, 59.5, 131.5, 255.5, 67.5, 60.5, 132.5, 255.5, 66.5, 61.5, 132.5, 255.5, 66.5, 62.5, 133.5, 255.5, 66.5, 64.5, 133.5, 255.5, 65.5, 65.5, 134.5, 255.5, 65.5, 66.5, 134.5, 255.5, 64.5, 67.5, 135.5, 255.5, 64.5, 68.5, 135.5, 255.5, 63.5, 69.5, 135.5, 255.5, 63.5, 71.5, 136.5, 255.5, 62.5, 72.5, 136.5, 255.5, 62.5, 73.5, 137.5, 255.5, 61.5, 74.5, 137.5, 255.5, 61.5, 75.5, 137.5, 255.5, 61.5, 76.5, 137.5, 255.5, 60.5, 77.5, 138.5, 255.5, 60.5, 78.5, 138.5, 255.5, 59.5, 80.5, 138.5, 255.5, 59.5, 81.5, 138.5, 255.5, 58.5, 82.5, 139.5, 255.5, 58.5, 83.5, 139.5, 255.5, 57.5, 84.5, 139.5, 255.5, 57.5, 85.5, 139.5, 255.5, 56.5, 86.5, 139.5, 255.5, 56.5, 87.5, 140.5, 255.5, 55.5, 88.5, 140.5, 255.5, 55.5, 89.5, 140.5, 255.5, 54.5, 90.5, 140.5, 255.5, 54.5, 91.5, 140.5, 255.5, 53.5, 92.5, 140.5, 255.5, 53.5, 93.5, 140.5, 255.5, 52.5, 94.5, 141.5, 255.5, 52.5, 95.5, 141.5, 255.5, 51.5, 96.5, 141.5, 255.5, 51.5, 97.5, 141.5, 255.5, 50.5, 98.5, 141.5, 255.5, 50.5, 99.5, 141.5, 255.5, 49.5, 100.5, 141.5, 255.5, 49.5, 101.5, 141.5, 255.5, 49.5, 102.5, 141.5, 255.5, 48.5, 103.5, 141.5, 255.5, 48.5, 104.5, 141.5, 255.5, 47.5, 105.5, 141.5, 255.5, 47.5, 106.5, 141.5, 255.5, 46.5, 107.5, 142.5, 255.5, 46.5, 108.5, 142.5, 255.5, 46.5, 109.5, 142.5, 255.5, 45.5, 110.5, 142.5, 255.5, 45.5, 111.5, 142.5, 255.5, 44.5, 112.5, 142.5, 255.5, 44.5, 113.5, 142.5, 255.5, 44.5, 114.5, 142.5, 255.5, 43.5, 115.5, 142.5, 255.5, 43.5, 116.5, 142.5, 255.5, 42.5, 117.5, 142.5, 255.5, 42.5, 118.5, 142.5, 255.5, 42.5, 119.5, 142.5, 255.5, 41.5, 120.5, 142.5, 255.5, 41.5, 121.5, 142.5, 255.5, 40.5, 122.5, 142.5, 255.5, 40.5, 122.5, 142.5, 255.5, 40.5, 123.5, 142.5, 255.5, 39.5, 124.5, 142.5, 255.5, 39.5, 125.5, 142.5, 255.5, 39.5, 126.5, 142.5, 255.5, 38.5, 127.5, 142.5, 255.5, 38.5, 128.5, 142.5, 255.5, 38.5, 129.5, 142.5, 255.5, 37.5, 130.5, 142.5, 255.5, 37.5, 131.5, 141.5, 255.5, 36.5, 132.5, 141.5, 255.5, 36.5, 133.5, 141.5, 255.5, 36.5, 134.5, 141.5, 255.5, 35.5, 135.5, 141.5, 255.5, 35.5, 136.5, 141.5, 255.5, 35.5, 137.5, 141.5, 255.5, 34.5, 137.5, 141.5, 255.5, 34.5, 138.5, 141.5, 255.5, 34.5, 139.5, 141.5, 255.5, 33.5, 140.5, 141.5, 255.5, 33.5, 141.5, 140.5, 255.5, 33.5, 142.5, 140.5, 255.5, 32.5, 143.5, 140.5, 255.5, 32.5, 144.5, 140.5, 255.5, 32.5, 145.5, 140.5, 255.5, 31.5, 146.5, 140.5, 255.5, 31.5, 147.5, 139.5, 255.5, 31.5, 148.5, 139.5, 255.5, 31.5, 149.5, 139.5, 255.5, 31.5, 150.5, 139.5, 255.5, 30.5, 151.5, 138.5, 255.5, 30.5, 152.5, 138.5, 255.5, 30.5, 153.5, 138.5, 255.5, 30.5, 153.5, 138.5, 255.5, 30.5, 154.5, 137.5, 255.5, 30.5, 155.5, 137.5, 255.5, 30.5, 156.5, 137.5, 255.5, 30.5, 157.5, 136.5, 255.5, 30.5, 158.5, 136.5, 255.5, 30.5, 159.5, 136.5, 255.5, 30.5, 160.5, 135.5, 255.5, 31.5, 161.5, 135.5, 255.5, 31.5, 162.5, 134.5, 255.5, 31.5, 163.5, 134.5, 255.5, 32.5, 164.5, 133.5, 255.5, 32.5, 165.5, 133.5, 255.5, 33.5, 166.5, 133.5, 255.5, 33.5, 167.5, 132.5, 255.5, 34.5, 167.5, 132.5, 255.5, 35.5, 168.5, 131.5, 255.5, 35.5, 169.5, 130.5, 255.5, 36.5, 170.5, 130.5, 255.5, 37.5, 171.5, 129.5, 255.5, 38.5, 172.5, 129.5, 255.5, 39.5, 173.5, 128.5, 255.5, 40.5, 174.5, 127.5, 255.5, 41.5, 175.5, 127.5, 255.5, 42.5, 176.5, 126.5, 255.5, 43.5, 177.5, 125.5, 255.5, 44.5, 177.5, 125.5, 255.5, 46.5, 178.5, 124.5, 255.5, 47.5, 179.5, 123.5, 255.5, 48.5, 180.5, 122.5, 255.5, 50.5, 181.5, 122.5, 255.5, 51.5, 182.5, 121.5, 255.5, 53.5, 183.5, 120.5, 255.5, 54.5, 184.5, 119.5, 255.5, 56.5, 185.5, 118.5, 255.5, 57.5, 185.5, 118.5, 255.5, 59.5, 186.5, 117.5, 255.5, 61.5, 187.5, 116.5, 255.5, 62.5, 188.5, 115.5, 255.5, 64.5, 189.5, 114.5, 255.5, 66.5, 190.5, 113.5, 255.5, 68.5, 190.5, 112.5, 255.5, 69.5, 191.5, 111.5, 255.5, 71.5, 192.5, 110.5, 255.5, 73.5, 193.5, 109.5, 255.5, 75.5, 194.5, 108.5, 255.5, 77.5, 194.5, 107.5, 255.5, 79.5, 195.5, 105.5, 255.5, 81.5, 196.5, 104.5, 255.5, 83.5, 197.5, 103.5, 255.5, 85.5, 198.5, 102.5, 255.5, 87.5, 198.5, 101.5, 255.5, 89.5, 199.5, 100.5, 255.5, 91.5, 200.5, 98.5, 255.5, 94.5, 201.5, 97.5, 255.5, 96.5, 201.5, 96.5, 255.5, 98.5, 202.5, 95.5, 255.5, 100.5, 203.5, 93.5, 255.5, 103.5, 204.5, 92.5, 255.5, 105.5, 204.5, 91.5, 255.5, 107.5, 205.5, 89.5, 255.5, 109.5, 206.5, 88.5, 255.5, 112.5, 206.5, 86.5, 255.5, 114.5, 207.5, 85.5, 255.5, 116.5, 208.5, 84.5, 255.5, 119.5, 208.5, 82.5, 255.5, 121.5, 209.5, 81.5, 255.5, 124.5, 210.5, 79.5, 255.5, 126.5, 210.5, 78.5, 255.5, 129.5, 211.5, 76.5, 255.5, 131.5, 211.5, 75.5, 255.5, 134.5, 212.5, 73.5, 255.5, 136.5, 213.5, 71.5, 255.5, 139.5, 213.5, 70.5, 255.5, 141.5, 214.5, 68.5, 255.5, 144.5, 214.5, 67.5, 255.5, 146.5, 215.5, 65.5, 255.5, 149.5, 215.5, 63.5, 255.5, 151.5, 216.5, 62.5, 255.5, 154.5, 216.5, 60.5, 255.5, 157.5, 217.5, 58.5, 255.5, 159.5, 217.5, 56.5, 255.5, 162.5, 218.5, 55.5, 255.5, 165.5, 218.5, 53.5, 255.5, 167.5, 219.5, 51.5, 255.5, 170.5, 219.5, 50.5, 255.5, 173.5, 220.5, 48.5, 255.5, 175.5, 220.5, 46.5, 255.5, 178.5, 221.5, 44.5, 255.5, 181.5, 221.5, 43.5, 255.5, 183.5, 221.5, 41.5, 255.5, 186.5, 222.5, 39.5, 255.5, 189.5, 222.5, 38.5, 255.5, 191.5, 223.5, 36.5, 255.5, 194.5, 223.5, 34.5, 255.5, 197.5, 223.5, 33.5, 255.5, 199.5, 224.5, 31.5, 255.5, 202.5, 224.5, 30.5, 255.5, 205.5, 224.5, 29.5, 255.5, 207.5, 225.5, 28.5, 255.5, 210.5, 225.5, 27.5, 255.5, 212.5, 225.5, 26.5, 255.5, 215.5, 226.5, 25.5, 255.5, 218.5, 226.5, 24.5, 255.5, 220.5, 226.5, 24.5, 255.5, 223.5, 227.5, 24.5, 255.5, 225.5, 227.5, 24.5, 255.5, 228.5, 227.5, 24.5, 255.5, 231.5, 228.5, 25.5, 255.5, 233.5, 228.5, 25.5, 255.5, 236.5, 228.5, 26.5, 255.5, 238.5, 229.5, 27.5, 255.5, 241.5, 229.5, 28.5, 255.5, 243.5, 229.5, 30.5, 255.5, 246.5, 230.5, 31.5, 255.5, 248.5, 230.5, 33.5, 255.5, 250.5, 230.5, 34.5, 255.5, 253.5, 231.5, 36.5, 255.5, 0.5, 0.5, 0.5, 255.5, 255.5, 255.5, 255.5, 255.5, 169.5, 169.5, 169.5, 255.5],
```

I made a gist to reproduce the problem:
https://gist.github.com/xavArtley/6165a6ab1f0f73b1e0fa4542fb479072

If I want to apply my lookup table I need to add this lines after the first render (commented in the gist):
```
//-------------------------------------------------------------------------------
// Need to change the table and update the render
//-------------------------------------------------------------------------------
lut.set({'table': actorJSON.mapper.lookupTable.table})
actor.modified();
renderWindow.render();
```

### PR DESCRIPTION
**Pull request by floryst**
Also support setting and getting individual table values.

Fixes #1257

### CODE REVIEW

File: Sources/Common/Core/LookupTable/index.js [hunk 1]
```diff
@@ -175,6 +177,66 @@ function vtkLookupTable(publicAPI, model) {
     } // alpha blending
   };
 
+  publicAPI.setTable = (newTable) => {
+    if (Array.isArray(newTable) && newTable !== model.table) {
+      if (newTable.length % 4 !== 0) {
+        vtkErrorMacro('setTable: table must have 4-component entries');
+        return false;
+      }
+      model.table = newTable;
```
Comment (floryst) [inline Sources/Common/Core/LookupTable/index.js]: Do we care if this is the same reference, or should I copy newTable?

Comment (jourdain) [reply Sources/Common/Core/LookupTable/index.js]: copy

**Comment by floryst** [comment]
Wait, I just realized I overlapped a bit on #1274. I'll wait for that to merge, and then I'll update this PR. Alternatively, @thewtex if you want to add logic for `set/getTableValue` and the initialization code [here](https://github.com/Kitware/vtk-js/pull/1301/files#diff-86b1a7ebf45bd0f69b5b4173b143e62aR330) to your PR, then I can close this one.

**Comment by thewtex** [comment]
@floryst would you mind review and merging #1274, then rebasing?

We could support passing an `Array` with this PR in addition to the `vtkDataArray` in #1274.

Regarding `getTable`, I think we need a function defined manually that excludes the "special colors" instead of directly returning the `table` member. We can then call `getTable` then `setTable` and have the same state, also we do not duplicate / overwrite setting state with the NaN color, etc.

**Comment by floryst** [comment]
I'm going to remove my `get/setTable` stuff since it's not in scope. I'll just focus on fixing #1257 in this PR.

**Pull request merged**